jobs/tendrl-jobs-build-triggers.yaml

```diff
@@ -0,0 +1,76 @@
+- job-template:
+    name: "tendrl-build-{package_name}"
+    display-name: 'Tendrl-{package_name}'
+    description: 'Do not edit this job through the web!'
+    project-type: freestyle
+    defaults: global
+    disabled: false
+    concurrent: false
+    quiet-period: 0
+    block-downstream: false
+    block-upstream: false
+    node: 'master'
+    triggers:
+        - pollscm:
+            cron: 'H/10 * * * *'
+            ignore-post-commit-hooks: false
+    scm:
+        - git:
+            url: 'https://github.com/Tendrl/{package_name}'
+            branches:
+                - "*/master"
+            skip-tag: true
+            wipe-workspace: true
+    builders:
+        - shell: |
+            #!/bin/bash -xe
+            RELEASE=$(date +"%m_%d_%Y_%H_%M_%S")
+            sed -i tendrl-{package_name}.spec -e "s/^Release: .*/Release: $RELEASE/"
+            rm -fr {src_dir}/tendrl-{package_name}*.src.rpm
+            {pre_build_command}
+            make dist
+            {build_srpm_command}
+            copr-cli build tendrl {src_dir}/tendrl-{package_name}*.src.rpm
+
+- project:
+    name: 'tendrl'
+    jobs:
+      - "tendrl-build-{package_name}"
+    package_name:
+      - api:
+          project_name: 'Api'
+          src_dir: '~/rpmbuild/SRPMS'
+          pre_build_command: ''
+          build_srpm_command: 'rpmbuild -bs tendrl-{package_name}.spec'
+      - ceph-integration:
+          project_name: 'Ceph-Integration'
+          src_dir: '.'
+          pre_build_command: ''
+          build_srpm_command: 'make srpm'
+      - commons:
+          project_name: 'Commons'
+          src_dir: '.'
+          pre_build_command: ''
+          build_srpm_command: 'make srpm'
+      - dashboard:
+          project_name: 'Dashboard'
+          src_dir: '.'
+          pre_build_command: |
+              npm install gulp
+              npm install
```

Comment:
We need just `npm` package, see https://github.com/Tendrl/dashboard/issues/141#issuecomment-304020717